### PR TITLE
pnpm lock file seems to be the problem and also you had two test scri…

### PR DESCRIPTION
**Pull request by techyNonso**
…pts on package.json

## Related Issue(s)

Fixes #
Resolves #
Closes #

## Proposed Changes

[//]: # '[List down the specific changes you have made in this pull request.]'

- [x]

## Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [ ] I have added suitable unit tests (if applicable).

## To-Do

- [ ]

### CODE REVIEW

File: src/app/dashboard/train/components/ModelTraining.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default function ModelTraining({ setPage }: IProps) {
       </div>
       <div className="absolute top-0 inset-0 leading-8 pt-8 ">
         <h1 className="text-center text-3xl font-logirentBold">
-          Model Is now Being TraIned
+          Model Is now BeIng TraIned
```
Comment (Nasfame) [inline src/app/dashboard/train/components/ModelTraining.tsx:34]: not correct

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: src/app/dashboard/train/components/ModelTraining.tsx (hunk 1 above)
Comment (techyNonso) [reply src/app/dashboard/train/components/ModelTraining.tsx:34]: What is not correct ?

Comment (techyNonso) [reply src/app/dashboard/train/components/ModelTraining.tsx:34]: Leave the text like that,  the font family works in a  funny way . Do to get it to display the text in all caps,  you have to write it like that

Comment (techyNonso) [reply src/app/dashboard/train/components/ModelTraining.tsx:34]: Check it on preview

Comment (techyNonso) [reply src/app/dashboard/train/components/ModelTraining.tsx:34]: I see they left it like that on figma so I'll revert ot to lower case and also revert it to lower case in every file where I have forcefully made it uppercase before

Comment (Nasfame) [reply src/app/dashboard/train/components/ModelTraining.tsx:34]: but why is it BeIng

Comment (Nasfame) [reply src/app/dashboard/train/components/ModelTraining.tsx:34]: ??

Comment (techyNonso) [reply src/app/dashboard/train/components/ModelTraining.tsx:34]: It's the style of the font I belive

**Review by Nasfame — APPROVED** [review]
LGTM

**Pull request merged**